code/modules/martial_arts/combos/sleeping_carp/gnashing_teeth.dm

```diff
@@ -10,7 +10,7 @@
 					"<span class='userdanger'>[user] [atk_verb] you!</span>")
 	playsound(get_turf(target), 'sound/weapons/punch1.ogg', 25, TRUE, -1)
 	add_attack_logs(user, target, "Melee attacked with martial-art [MA] : Gnashing Teeth", ATKLOG_ALL)
-	target.apply_damage(20, BRUTE, user.zone_selected)
+	target.apply_damage(20, BRUTE, user.zone_selected, 0, TRUE)
```

Comment:
```suggestion
	target.apply_damage(20, BRUTE, user.zone_selected, 0, sharp = TRUE)
```
might aswell indicate what var this is for readability sake

Reply:
There no need to add the 0 Param before aswell if you use named parameters